### PR TITLE
RBMHUIM-2504: PHP: Find a user

### PR DESCRIPTION
**Pull request by mteufner**

### CODE REVIEW

File: php/README.md [hunk 1]
```diff
@@ -9,6 +9,13 @@ File: `client-form-submission.php`
 
 REST documentation: https://uim.redbull.com/uim/resources/docs/rest-api.html#client-resources-form-submissions-create
 
+### 3. Client Find a User
```
Comment (PatrickSchuster) [inline php/README.md]: Shouldn't that be nr. 2?

File: php/README.md [hunk 2]
```diff
@@ -9,6 +9,13 @@ File: `client-form-submission.php`
...
+Finds a user using the Client API.
+
+File: `client-find-user.php`
+
+REST documentation: https://uim.redbull.com/uim/resources/docs/rest-api.html#client-resources-user-get-user
```
Comment (PatrickSchuster) [inline php/README.md]: I think this also requires a docker documentation or we should make it generic (not use `php-form-submission` for the container name)?

File: php/README.md (hunk 1 above)
Comment (mteufner) [reply php/README.md]: https://github.com/redbullmediahouse/uim-integration-examples/pull/13 is number 2.

File: php/README.md (hunk 2 above)
Comment (mteufner) [reply php/README.md]: That I already did in the previous PR https://github.com/redbullmediahouse/uim-integration-examples/pull/13.

Comment (PatrickSchuster) [reply php/README.md]: Oh, ok! Thanks!

**Pull request merged**